### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by lgorence** [mysticlgbt/abbrv#1]
Getting started documentation would be a nice to have, you'd need to read the code right now to do so.

**Comment by raphaelsiz** [comment]
I agree, I even thought of doing a pull request to help but honestly reading the code I can't quite tell how it's supposed to be used
